site/profile/manifests/platform/monitoring/splunkforwarder.pp

```diff
@@ -0,0 +1,38 @@
+# Splunk Forwarder class
+class profile::platform::monitoring::splunkforwarder (
+  Optional[String] $splunk_server = undef,
+){
+
+  # if someone doesn't specify a splunk_server as a param, query PuppetDB
+  # This query grabs the host information for the Splunk server and creates an Exported Host Resource.
+
+  if $splunk_server == undef {
+    $splunk_nodes_query = 'resources[certname] { type = "Class" and title = "Splunk" }'
+    $_splunk_server = puppetdb_query($splunk_nodes_query)[0][certname]
+    Host  <<| tag == 'splunkserver' |>>
+  } else {
+    $_splunk_server = $splunk_server
+  }
+
+  class { '::splunk::params':
+    server   => $_splunk_server,
+    src_root => 'splunk::params::src_root',
```

Comment:
Don't think this src_root value works. You're setting it to the literal string 'splunk::params::src_root'. Is it supposed to be $splunk::params:src_root or ::splunk::params:src_root?